Tasks/Doroshevich/Task4/Task4.py

```diff
@@ -0,0 +1,44 @@
+while True:
+    n = input("Select the number of characters per line(min - 36):\n")
+    if n.isnumeric() and int(n) > 35:
+        n = int(n)
+        break
+    else:
+        print("Wrong data. Please try again.\n")
+
+with open("text.txt", 'r') as f:
+    with open(f"text_{n}_symbols.txt", 'w') as new:
+        for line in f:
```

Comment:
not the most optimal, since the file can contain no line ends, but still good approach